woocommerce/woocommerce-ios
# Update header stack view spacing - fixes wrong separator padding in dashboard banner

Closes: #10819


## Description


- In the dashboard, when the banner "We couldn't load your data" appear (for example when a trial expired), the bottom separator under the banner it's wrongly positioned (4pt above the white border). [See original issue](https://github.com/woocommerce/woocommerce-ios/issues/10819)

<img src="https://github.com/woocommerce/woocommerce-ios/assets/495617/a85ebb2a-55ac-4046-be25-16b1fd0fd394" width=300 />

- The fix itself was straightforward, just changing the value of the spacing constant from 4 to 0

## Testing instructions


- For a proper test try to get in the state when your trial expires, open the dashboard, check that the top error banner view is shown and check that now there is no spacing between the separator and that light gray view (as shown in the screenshots)
- If you are having trouble getting that error state, open `StoreStatsAndTopPerformersViewController.swift` and before line 169 (`if let error = syncError`) add this: `syncError = SiteStatsStoreError.unknown` which will fake the error and show the top banner view

## Screenshots


| Before      | After |
| ----------- | ----------- |
| ![Simulator Screenshot - iPhone 15 Pro - 2024-01-05 at 14 16 19 copy](https://github.com/woocommerce/woocommerce-ios/assets/6242034/4f9ff536-5c0b-42cf-b1f1-65d7545a03b4)   | ![Simulator Screenshot - iPhone 15 Pro - 2024-01-05 at 14 14 43 copy](https://github.com/woocommerce/woocommerce-ios/assets/6242034/343a1dee-30a8-447a-b9a7-7ae0f1cdbaf3) |

---
